app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  def javascript_include_env_tag(asset, dev_asset, params = {})
```

Comment:
This name is too close to the default rails one `javascript_include_tag` and thus easy to miss.

I'd suggest `env_specific_javascript_include_tag` and `env_specific_stylesheet_link_tag`.